posts/2024-01-30-24.0.0.1.adoc

```diff
@@ -0,0 +1,116 @@
+
+Check out link:{url-prefix}/blog/?search=release&search!=beta[previous Open Liberty GA release blog posts].
+
+
+[#run]
+
+== Develop and run your apps by using 24.0.0.1
+
+If you're using link:{url-prefix}/guides/maven-intro.html[Maven], include the following code in your `pom.xml` file:
+
+[source,xml]
+----
+<plugin>
+    <groupId>io.openliberty.tools</groupId>
+    <artifactId>liberty-maven-plugin</artifactId>
+    <version>3.10</version>
+</plugin>
+----
+
+Or for link:{url-prefix}/guides/gradle-intro.html[Gradle], include the following in your `build.gradle` file:
+
+[source,gradle]
+----
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'io.openliberty.tools:liberty-gradle-plugin:3.8'
+    }
+}
+apply plugin: 'liberty'
+----
+
+Or if you're using link:{url-prefix}/docs/latest/container-images.html[container images]:
+
+[source]
+----
+FROM icr.io/appcafe/open-liberty
+----
+
+Or take a look at our link:{url-prefix}/start/[Downloads page].
+
+If you're using link:https://plugins.jetbrains.com/plugin/14856-liberty-tools[IntelliJ IDEA], link:https://marketplace.visualstudio.com/items?itemName=Open-Liberty.liberty-dev-vscode-ext[Visual Studio Code] or link:https://marketplace.eclipse.org/content/liberty-tools[Eclipse IDE], you can also take advantage of our open source link:https://openliberty.io/docs/latest/develop-liberty-tools.html[Liberty developer tools] to enable effective development, testing, debugging, and application management all from within your IDE.
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // //
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/27303
+// Contact/Reviewer: anjumfatima90
+// // // // // // // //
+[#instanton_features]
+== Instant On support for jaxws/xmlws and other features
+
+link:https://openliberty.io/docs/latest/instanton.html[Open Liberty InstantOn] provides fast startup times for MicroProfile and Jakarta EE applications.
+
+InstantOn supports link:https://openliberty.io/docs/latest/instanton.html#supported-features[a subset of Open Liberty features]. Any public features that are enabled outside of this set of supported features cause the checkpoint to fail with an error message. As of the 24.0.0.1 release, the following features are enhanced to support InstantOn, in addition to the set of features included initial Liberty InstantOn 23.0.0.6 release:
+
+- `jaxws-2.2` (24.0.0.1 and later)
+- `xmlWS-3.0` (24.0.0.1 and later)
+- `xmlWS-4.0` (24.0.0.1 and later)
+- `jdbc-4.1` (23.0.0.9 and later)
+- `jdbc-4.3` (23.0.0.9 and later)
+- `appSecurity-1.0` (23.0.0.10 and later)
+- `javaMail-1.6` (23.0.0.10 and later)
+- `mail-2.0` (23.0.0.10 and later)
+- `mail-2.1` (23.0.0.10 and later)
+- `passwordUtilities-1.0` (23.0.0.10 and later)
+- `localConnector-1.0` (23.0.0.11 and later)
```

Comment:
I'm fine with keeping this as-is, but we can also consider using the full names for the features, for example:
- Java Web Services 2.2 (`jaxws-2.2`) - 24.0.0.1 and later
- Jakarta XML Web Services (`xmlWS-3.0`) -  24.0.0.1 and later
.....

Another option would be to make this into a 3 column table: Name, Feature, Initial InstantOn support

Reply:
added the table and security tag